DFC.App.JobProfile.CurrentOpportunities/Controllers/SegmentController.cs

```diff
@@ -69,6 +68,30 @@ public async Task<IActionResult> Document(string article)
             return NoContent();
         }
 
+        [HttpGet]
+        [Route("segment/simplelist")]
+        public async Task<IActionResult> SimpleList()
+        {
+            logger.LogInformation($"{nameof(SimpleList)} has been called");
+
+            var viewModel = new SimpleListViewModel();
+            var currentOpportunitiesSegmentModels = await currentOpportunitiesSegmentService.GetAllAsync().ConfigureAwait(false);
+
+            if (currentOpportunitiesSegmentModels != null)
+            {
+                viewModel.Items = (from a in currentOpportunitiesSegmentModels.OrderBy(o => o.CanonicalName)
+                                   select mapper.Map<SimpleDocumentViewModel>(a)).ToList();
+
+                logger.LogInformation($"{nameof(SimpleList)} has succeeded");
+            }
+            else
+            {
+                logger.LogWarning($"{nameof(SimpleList)} has returned with no results");
+            }
+
+            return this.NegotiateContentResult(viewModel, viewModel.Items);
+        }
```

Comment:
Do we need this? Can we not use the default index action and add content negotiation to that?

Reply:
removed as requested, now using "Index" action instead